Szwalnia/Informacje o zatrudnieniu.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace Szwalnia
+{
+    public partial class InformacjeOZatrudnieniu : Form
+    {
+        public SzwalniaEntities db;
+        public InformacjeOZatrudnieniu(SzwalniaEntities db)
+        {
+            InitializeComponent();
+            this.db = db;
+            
+            cbUmowa.DataSource = db.Rodzaj_Umowy.ToList();
+            cbUmowa.ValueMember = "ID_Rodzaj_Umowy";
+            cbUmowa.DisplayMember = "Rodzaj_Umowy";
+
+
+            cbImieNazwisko.DataSource = db.Pracownicy.ToList();
+            cbImieNazwisko.ValueMember = "ID_Pracownika";
+            cbImieNazwisko.DisplayMember = "Imie"; //'Pracownik'
```

Comment:
imię to słaby kandydat - conajmniej nazwisko a moim zdaniem powinno to być imię i nazwisko (wtedy potrzebny widok, który będzie miał takie pole)

Reply:
rzeczywiście przez widok wyszło bardzo dobrze.